### PR TITLE
Add initial codeowners, gitignore, contributing files

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@lizthegrey can you pls enable EasyCLA here?

**Comment by bogdandrutu** [comment]
Will do

**Comment by bogdandrutu** [comment]
@lizthegrey I thought will automatically check for EasyCLA seems that I was wrong, anything specifically that I need to do - it does not look like available in the status checks.

**Comment by bogdandrutu** [comment]
@SergeyKanzhelev done, please approve now :)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#####################################################
+#
+# List of approvers for OpenTelemetry Proto repository
```
Comment (tigrannajaryan) [inline .github/CODEOWNERS:3]:
```suggestion
# List of approvers for OpenTelemetry Build repository
```

**Pull request merged**